### ISSUE TITLE
[Beta SDK 42] Expo-contacts throws SAXParseException

### ISSUE THREAD

**Issue by sshquack** [expo/expo#13407]
### Summary

While trying to verify the bug fix from https://github.com/expo/expo/issues/13100 using the latest Expo beta 42, expo-contacts throws `SAXParseException`.

```
* What went wrong:
Execution failed for task ':expo-contacts:processDebugManifest'.
> Failed to calculate the value of task ':expo-contacts:processDebugManifest' property 'packageOverride'.
   > Failed to query the value of property 'applicationId'.
      > org.xml.sax.SAXParseException; systemId: file:/Users/foo/Desktop/code/expo/boring-app/node_modules/expo-contacts/android/src/main/AndroidManifest.xml; lineNumber: 4; columnNumber: 65; The prefix "android" for attribute "android:name" associated with an element type "action" is not bound.
```

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

Android

### SDK Version (managed workflow only)

42 Beta

### Environment

```
❯ expo diagnostics

  Expo CLI 4.7.1 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 14.17.0 - ~/.n/bin/node
      Yarn: 1.22.10 - ~/.n/bin/yarn
      npm: 7.19.0 - ~/.n/bin/npm
      Watchman: 2021.06.07.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.10.1 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    IDEs:
      Xcode: 12.4/12D4e - /usr/bin/xcodebuild
    npmPackages:
      expo: next => 42.0.0-beta.1
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: 0.63.4 => 0.63.4
      react-native-web: ~0.13.12 => 0.13.18
    npmGlobalPackages:
      expo-cli: 4.7.1
    Expo Workflow: bare

```

### Reproducible demo or steps to reproduce from a blank project

- Same as https://github.com/expo/expo/issues/13100
- Repro project is at https://github.com/sshquack/boring-app-01

**Comment by byCedric** [comment]
Hi @sshquack! Thanks for the report, I believe we fixed this issue in #13399. If you can install `expo-contacts@9.2.4` you should be able to build!

**Comment by sshquack** [comment]
@byCedric Great to hear its fixed. I upgraded to 42-beta2 and I see a new error https://github.com/expo/expo/issues/13427
